src/css/base.scss

```diff
@@ -1945,6 +1949,10 @@ textarea {
     min-width: 48px;
 }
 
+.mw-800 {
```

Comment:
mw is confusing if it is for min or max can we change it to `min-w-800` like line number 2018

Reply:
done